### PR TITLE
[kube-prometheus-stack]  Scope CoreDNS dashboard to CoreDNS job

### PR DESCRIPTION
**Pull request by aslafy-z**
#### What this PR does / why we need it:

This PR scopes the metrics shown in the CoreDNS dashboard to the CoreDNS job metrics (excluding node local dns)

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 30.0.2
+version: 30.0.3
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:24]:
```suggestion
version: 30.0.4
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (aslafy-z) [reply charts/kube-prometheus-stack/Chart.yaml]: I just bumped the version

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 30.0.3
+version: 30.0.4
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:24]:
```suggestion
version: 30.1.1
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Comment by desaintmartin** [comment]
Thanks! Could you rebase?

**Review by desaintmartin — APPROVED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**